### ISSUE TITLE
Publish the extension to open-vsx.org too

### ISSUE THREAD

**Issue by jakobbouchard** [Shopify/theme-check-vscode#80]
## Feature request summary

It would be great to also have it on [open-vsx.org](https://open-vsx.org/), as it is an open platform.

## Rationale

It would make it available to people using VSCodium, Gitpod, etc. It would make for a better experience for more developers, and it would also support open platforms.

**Comment by Elliottmeija** [comment]
Bump. Really need this as there aren't any liquid extensions for Gitpod. I'm considering just making a port...

**Comment by jakobbouchard** [comment]
@charlespwd Yeah I can understand that. If we open a PR or an issue in the open-vsx repo for them to add it, would that work with you?

Also just hijacking this thread to ask if [my issue](https://github.com/Shopify/quilt/issues/2418) in Shopify/quilt was seen, as well as [this issue](https://community.shopify.com/c/shopify-apps/your-embedded-app-is-redirecting-the-top-frame-outside-of-the/m-p/1730903) with submissions that affects multiple people but hasn't been acknowledged (as far as I know) by any Shopify staff...

**Comment by charlespwd** [comment]
> If we open a PR or an issue in the open-vsx repo for them to add it, would that work with you?

I can't adequately judge if that would incur overhead on our part.

But, if it's not done in Shopify's name, this project is under the MIT license, so you are free to do so :upside_down_smile: (with proper attributions, of course)

**Comment by jakobbouchard** [comment]
As far as I know, it wouldn't cause overhead as open-vsx builds the extension themselves when we open that PR. I'll open a PR on their repo some time this week :) I'll keep the issue open until I've done so, then I'll close it to move the tracking to the PR.
